### PR TITLE
add support for manual region specification

### PR DESCRIPTION
**Pull request by nerdDan**

### CODE REVIEW

File: src/main/java/pro/apphub/aws/cloudwatch/log4j2/CloudWatchAppender.java [hunk 1]
```diff
@@ -90,6 +92,7 @@ public CloudWatchAppender(String name,
             this.group = group;
             this.stream = initStream(streamPrefix, streamPostfix);
             this.client = initClient(access, secret);
+            this.client.setRegion(RegionUtils.getRegion(region));
```
[the remark this answers is not in the document]
Comment (nerdDan) [reply src/main/java/pro/apphub/aws/cloudwatch/log4j2/CloudWatchAppender.java]: Actually I tried to do that at first place. But AWS Java SDK is confusing to me. I'll look into document later.

[the remark this answers is not in the document]
Comment (nerdDan) [reply src/main/java/pro/apphub/aws/cloudwatch/log4j2/CloudWatchAppender.java]: I mean, I was trying to specify the region through the constructor parameter of `AWSLogsClient`.
Doing `setRegion` inside `initClient` is easy enough. I guess I'll have to do so.

[the remark this answers is not in the document]
Comment (nerdDan) [reply src/main/java/pro/apphub/aws/cloudwatch/log4j2/CloudWatchAppender.java]: I guess this is what you mean.
```java
    private static AWSLogsClient initClient(String region, String access, String secret) {
        if ((access != null) && (secret != null)) {
            return new AWSLogsClient(new BasicAWSCredentials(access, secret));
        } else {
            AWSLogsClient client = new AWSLogsClient();
            client.setRegion(RegionUtils.getRegion(region));
            return client;
        }
    }
```
When I use this package outside an EC2 instance, but with the credentials specifying through AWS CLI, I don't have to assign `access` and `secret` in `log4j2.xml`. In this case, the above snippet does not work. Therefore, I suggest this.
```java
    private static AWSLogsClient initClient(String region, String access, String secret) {
        AWSLogsClient client;
        if ((access != null) && (secret != null)) {
            client = new AWSLogsClient(new BasicAWSCredentials(access, secret));
        } else {
            client = new AWSLogsClient();
        }
        if (region != null) {
            client.setRegion(RegionUtils.getRegion(region));
        }
        return client;
    }
```

[the remark this answers is not in the document]
Comment (apphub-pro) [reply src/main/java/pro/apphub/aws/cloudwatch/log4j2/CloudWatchAppender.java]: Yes, that's it.
And please create another pull request without existent param names modification.

**Comment by apphub-pro** [comment]
Now looks fine, thank you.

**Pull request merged**